pyvrp/cpp/ProblemData.h

```diff
@@ -18,6 +18,7 @@ namespace pyvrp
  *     vehicle_types: List[VehicleType],
  *     distance_matrix: List[List[int]],
  *     duration_matrix: List[List[int]],
+ *     groups: List[ClientGroup] = [],
```

Comment:
Nit: we can type these now as `list[...]` right?

Reply:
Sí!